### ISSUE TITLE
Highlight active view in navbar

### ISSUE THREAD

**Issue by egeldenhuys** [TripleParity/docks-ui#103]
It would be useful to see which tab the view is currently on

**Comment by FJMentz** [comment]
@egeldenhuys Yes, the navbar as it is now was just to get a final project asap, the nav still has a lot of changes that needs to be applied. Thank you for the feedback tho😁
